### PR TITLE
Fix for area light baked data not updating when changing the gizmo

### PR DESCRIPTION
**Pull request by FrancescoC-unity**
When editing the area light shape anywhere but via manually inserting value in the inspector, the legacy light size counterpart was not updated. 

This fixes that, therefore ending up fixing https://fogbugz.unity3d.com/f/cases/1244398/

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/Editor/Lighting/HDLightUI.Handles.cs [hunk 1]
```diff
@@ -662,6 +662,7 @@ public static void DrawHandles(HDAdditionalLightData additionalData, Editor owne
                                     {
                                         additionalData.shapeHeight = widthHeight.y;
                                     }
+
```
Comment (FrancescoC-unity) [inline com.unity.render-pipelines.high-definition/Editor/Lighting/HDLightUI.Handles.cs:665]: gonna revert this

**Comment by sebastienlagarde** [comment]
Merging this fix as it is small and Francesco test is enough

**Comment by RemyUnity** [comment]
> Merging this fix as it is small and Francesco test is enough

Please document in the PR what has been tested in that case next time. @FrancescoC-unity  @sebastienlagarde

**Comment by FrancescoC-unity** [comment]
> > Merging this fix as it is small and Francesco test is enough
> 
> Please document in the PR what has been tested in that case next time. @FrancescoC-unity @sebastienlagarde

In cases like this, I think is fairly clear that the only testing was the repro case for the bug.

**Pull request merged**